### PR TITLE
(CLOUD-1525) new features for subnet vnet and community fixes

### PR DESCRIPTION
**Pull request by gregohardy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1225,6 +1225,99 @@ The URI of a file containing the params in Azure Resource Model standard format.
...
 The format of this file differs from the format accepted by the `params` attribute. Must not be specified when `params` is specified.
 
+
+#### Type: azure_vnet
+
+##### `ensure`
+
+Specifies the basic state of the virtual network.
+
+Values: 'present', 'absent'.
```
Comment (EamonnTP) [inline README.md]: Values: `present`, `absent`.

File: README.md [hunk 2]
```diff
@@ -1225,6 +1225,99 @@ The URI of a file containing the params in Azure Resource Model standard format.
...
+
+Default: 'present'.
```
Comment (EamonnTP) [inline README.md]: Default: `present`.

File: README.md [hunk 3]
```diff
@@ -1225,6 +1225,99 @@ The URI of a file containing the params in Azure Resource Model standard format.
...
+tags => {'department' => 'devops', 'foo' => 'bar'}
+```
+
+#### Type: azure_subnet
+
+##### `ensure`
+
+Specifies the basic state of the subnet.
+
+Values: 'present', 'absent'.
```
Comment (EamonnTP) [inline README.md]: Values: `present`, `absent`.

File: README.md [hunk 4]
```diff
@@ -1225,6 +1225,99 @@ The URI of a file containing the params in Azure Resource Model standard format.
...
+
+Default: 'present'.
```
Comment (EamonnTP) [inline README.md]: Default: `present`.

**Comment by MWilsonPuppet** [comment]
LGTM

**Pull request merged**